consensus/src/liveness/proposal_generator_test.rs

```diff
@@ -9,8 +9,13 @@ use crate::{
 };
 use consensus_types::block::{block_test_utils::certificate_for_genesis, Block};
 use diem_types::validator_signer::ValidatorSigner;
+use futures::{future::BoxFuture, FutureExt};
 use std::sync::Arc;
 
+fn empty_callback() -> BoxFuture<'static, ()> {
+    async move {}.boxed()
```

Comment:
@bmwill, is this idiomatic?

Reply:
I mean its odd looking for sure but its perfectly ok way of creating a no-op future